Superbuild/OsprayExternal.cmake

```diff
@@ -34,7 +35,8 @@ LIST(APPEND ospray_DEPENDENCIES GLM_external)
 # git checkout -q will silence message about detached head (harmless).
 ExternalProject_Add(Ospray_external
   DEPENDS ${ospray_DEPENDENCIES}
-  GIT_REPOSITORY "https://github.com/CIBC-Internal/ospray.git"
+  # GIT_REPOSITORY "https://github.com/CIBC-Internal/ospray.git"
+  GIT_REPOSITORY "https://github.com/tarkpate/ospray.git"
```

Comment:
I suggest to maintain official changes associated with `SCIRun` into a fork of `ospray` added to the https://github.com/SCIInstitute organization.

For guidance on naming convention for the branches et alike.

See these examples:
* https://github.com/search?q=topic%3Akitware-fork+org%3AKitwareMedical+fork%3Atrue&type=repositories

Reply:
The CIBC-Internal repo is a part of the SCI Institute. There is a [PR](https://github.com/CIBC-Internal/ospray/pull/7) to put this branch in the OSPRay for there. Using my repo's branch is temporary for testing purposes.

Reply:
Sounds great. Thanks for the follow-up and clarification.